src/blueprints/tv2/timeline-object-factories/interfaces/tv2-audio-timeline-object-factory.ts

```diff
@@ -3,9 +3,10 @@ import { Tv2BlueprintConfiguration } from '../../value-objects/tv2-blueprint-con
 import { Tv2VideoClipManifestData } from '../../value-objects/tv2-action-manifest-data'
 import { Tv2SourceMappingWithSound } from '../../value-objects/tv2-studio-blueprint-configuration'
 import { DeviceType } from '../../../../model/enums/device-type'
+import { Tv2AudioMode } from '../../enums/tv2-audio-mode'
 
 export interface Tv2AudioTimelineObjectFactory {
-  createTimelineObjectsForSource(configuration: Tv2BlueprintConfiguration, source: Tv2SourceMappingWithSound): TimelineObject[]
+  createTimelineObjectsForSource(configuration: Tv2BlueprintConfiguration, source: Tv2SourceMappingWithSound, audioMode?: Tv2AudioMode): TimelineObject[]
```

Comment:
Why is `audioMode` optional?

Reply:
It's far more common to not use VoiceOver. Since `AudioMode` is used to indicate whether or not it should be VoiceOver, I thought nice not having to include everytime we make TimelineObjects for a Source when we don't care for VoiceOver.

I don't have a strong argument to keep it optional, so if it's preferred that it isn't optional, I'm up for it :)